src/Components/ImagesTable/ImageLinkDirect.js

```diff
@@ -101,19 +103,40 @@ const ImageLinkDirect = (props) => {
         </Popover>
       );
     } else if (uploadStatus.type === 'aws.s3') {
-      return (
-        <Button
-          component="a"
-          target="_blank"
-          variant="link"
-          icon={<DownloadIcon />}
-          iconPosition="right"
-          isInline
-          href={uploadStatus.options.url}
-        >
-          Download {fileExtensions[props.imageType]}
-        </Button>
-      );
+      if (!props.isExpired) {
+        return (
+          <Button
+            component="a"
+            target="_blank"
+            variant="link"
+            icon={<DownloadIcon />}
+            iconPosition="right"
+            isInline
+            href={uploadStatus.options.url}
+          >
+            Download {fileExtensions[props.imageType]}
+          </Button>
+        );
+      } else {
+        return (
+          <Button
+            component="a"
+            target="_blank"
+            variant="link"
+            onClick={() =>
+              navigate('/imagewizard', {
```

Comment:
This needs to be `navigate(resolveRelPath('/imagewizard'),...` due to https://github.com/RedHatInsights/image-builder-frontend/pull/833

Reply:
Got it, fixed :)